configuration/concepts/concepts_base.csv

```diff
@@ -0,0 +1,15 @@
+Uuid,Void/Retire,Fully specified name:en,Fully specified name:fr,Short name:en,Short name:fr,Description:en,Description:fr,Data class,Data type,_version:1,_order:1000
+0aca2d79-1a16-42c8-b2c0-a4989bd55aec,,AZT+3TC+ATZ/r,,AZT+3TC+ATZ/r,,AZT+3TC+ATZ/r,,Drug,N/A,,
```

Comment:
@IipperMangesh the import failed on the above concept (see below the error message from the log); I suspect this is due to the double commas

`
ERROR - CsvParser.saveAll(184) |2019-04-30 07:44:09,986| An OpenMRS object could not be constructed or saved from the following CSV line: [0aca2d79-1a16-42c8-b2c0-a4989bd55aec, null, AZT+3TC+ATZ/r, null, AZT+3TC+ATZ/r, null, AZT+3TC+ATZ/r, null, Drug, N/A, null, null]
`

Reply:
@IipperMangesh the concept names should be repeated 4 times on each row (full specified name in fr and en, and short name in fr and en); they are currently repeated 3 times